src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Border/Border_CornerRadius.xaml

```diff
@@ -0,0 +1,74 @@
+﻿<Page x:Class="Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls.Border_CornerRadius"
+	  xmlns:controls="using:Uno.UI.Samples.Controls"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:Uno.UI.Samples.UITests.BorderTestsControl"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:ios="http://uno.ui/ios"
+	  xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:android="http://uno.ui/android"
+	  mc:Ignorable="d ios android">
```

Comment:
local and controls xmlns should be removed or updated, since they exist in assembly that wont be referenced by RuntimeTests:
```suggestion
	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
	  mc:Ignorable="d">
```
unused win/ios/android is w/e

Reply:
Really good to know.